utils/watchcat/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=watchcat
 PKG_VERSION:=1
-PKG_RELEASE:=11
+PKG_RELEASE:=13
```

Comment:
should be 12, no?

Reply:
Fixed, my mistake. I had already bumped the version in my testing and had forgotten about that.